Docs/source/usage/parameters.rst

```diff
@@ -2385,16 +2385,22 @@ Reduced Diagnostics
         * ``<reduced_diags_name>.reduction_type`` (`string`)
             The type of reduction to be performed. It must be either ``Maximum``, ``Minimum`` or
             ``Integral``.
-            ``Integral`` computes the spatial integral of the function defined in the parser by
-            summing its value on all grid points and multiplying the result by the volume of a
-            cell.
+            ``Integral`` computes the spatial surface or volume integral, depending on the choice
+            of the integration_type, of the function defined in the parser by summing its value on
```

Comment:
```suggestion
            of the ``integration_type``, of the function defined in the parser by summing its value on
```

Reply:
The new documentation should ideally come under a new Type called 
`RawEFieldReduction` unlike, where it is now, FieldReduction.

The main different here FieldReduction first interpolates fields to the cell-center, where RawEFieldReduction integrates the fields from the staggered yee-grid